react/features/prejoin/components/Prejoin.native.tsx

```diff
@@ -85,7 +84,12 @@ const Prejoin: React.FC<PrejoinProps> = ({ navigation }: PrejoinProps) => {
 
     const onJoinLowBandwidth = useCallback(() => {
         dispatch(setAudioOnly(true));
-        onJoin();
+        dispatch(connect());
+        if (startCarmode) {
+            navigateRoot(screen.conference.carmode);
```

Comment:
This is not what we want, we want to go into the conference, and THEN enter car mode, so the user can then close it if they want to.